### PR TITLE
fix: remove play services dependency from fdroid flavor

### LINKED ISSUE #381: Remove play services dependency from the fdroid flavor
working on it
parent issue #367

### PR DESCRIPTION
**Pull request by nikit19**
fixes #381 
Removes acknowledgements from fdroid flavor

### CODE REVIEW

File: app/src/fdroid/java/org/fossasia/openevent/general/settings/SettingsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                        settingsViewModel.logout()
+                        startActivity(Intent(context, MainActivity::class.java))
+                        activity?.finish()
+                    }
+                }
+                .setNegativeButton(resources.getString(R.string.cancel)) { dialog, _ -> dialog.cancel() }
+        val alert = builder.create()
+        alert.show()
+    }
+}
```
Comment (iamareebjamal) [inline app/src/fdroid/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: Not a good idea to copy whole fragment, just encapsulate the changes in one component and add two versions of that component

**Comment by nikit19** [comment]
@iamareebjamal I thought of that earlier but
`import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
`
This import statement will cause a problem in fdroid build because this library won't be there.

**Comment by nikit19** [comment]
@iamareebjamal I think I got your point

**Comment by nikit19** [comment]
@iamareebjamal PR updated. Please review

File: app/src/fdroid/java/org/fossasia/openevent/general/settings/AcknowledgementDecider.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.fossasia.openevent.general.settings
+
+import android.content.Context
+
+object AcknowledgementDecider {
+
+    fun decide(context: Context) {
+       // don't show acknowledgements in fdroid
+    }
+}
```
Comment (iamareebjamal) [inline app/src/fdroid/java/org/fossasia/openevent/general/settings/AcknowledgementDecider.kt]: Add a method called `shouldShowAcknowledgement` and make it return true and false in the separate component

And use it to hide the acknowledgement

**Comment by nikit19** [comment]
@iamareebjamal PR updated. Please review.
Changed the value in strings of XML as "About" was already there in strings

**Pull request merged**